roles/setup_pgd/defaults/main.yml

```diff
@@ -111,5 +111,28 @@ supported_pg_version:
   - 14
   - 15
 
+# The following parameters must be configured in the 'pgd_commit_scopes' list
+# the values below are examples and should be adjusted to your requirements
+#  for CAMO
+#  - cs_type: 'CAMO'
+#  - cs_name: 'camo_scope_1'
+#  - parent_group: 'pgd_cluster'
+#  - cs_origin_node_group: 'pgd_two_nodes'
+#  - member_nodes: ['edb-primary1', 'edb-primary2']
+#  - default_group_cs: true
+#  - cs_rule: "ALL ( pgd_two_nodes ) ON visible CAMO DEGRADE ON (timeout=500s) TO ASYNC"
+#  for GROUP_COMMIT
+#  - cs_type: 'GROUP_COMMIT'
+#  - cs_name: 'group_commit_scope_1'
+#  - parent_group: 'pgd_cluster'
+#  - cs_origin_node_group: 'pgd_gc_nodes'
+#  - member_nodes: ['edb-primary3']
+#  - default_group_cs: true
+#  - cs_rule: "ALL ( pgd_remaining_nodes ) GROUP COMMIT"
 supported_pgd_version:
   - 5
+
+# For CAMO Commit Scopes
+max_prepared_transactions: 100
+
+pgd_commit_scopes: ""
```

Comment:
these should also be with the comments and I think they should be moved up some more
```
# The following parameters must be configured in the 'pgd_commit_scopes' list
# the values below are examples and should be adjusted to your requirements
#  for CAMO
# pgd_commit_scopes:
#  - cs_type: 'CAMO'
#  - cs_name: 'camo_scope_1'
#  - parent_group: 'pgd_cluster'
#  - cs_origin_node_group: 'pgd_two_nodes'
#  - member_nodes: ['edb-primary1', 'edb-primary2']
#  - default_group_cs: true
#  - cs_rule: "ALL ( pgd_two_nodes ) ON visible CAMO DEGRADE ON (timeout=500s) TO ASYNC"
#  for GROUP_COMMIT
# pgd_commit_scopes:
#  - cs_type: 'GROUP_COMMIT'
#  - cs_name: 'group_commit_scope_1'
#  - parent_group: 'pgd_cluster'
#  - cs_origin_node_group: 'pgd_gc_nodes'
#  - member_nodes: ['edb-primary3']
#  - default_group_cs: true
#  - cs_rule: "ALL ( pgd_remaining_nodes ) GROUP COMMIT"
pgd_commit_scopes: ""

# when CAMO commit scope present, max_prepared_transactions cannot be 0 (default)
max_prepared_transactions: 100

etc_hosts_lists: []
supported_os:
  - CentOS7
  - CentOS8
  - RedHat7
  - RedHat8
  - Rocky8
  - AlmaLinux8
  - OracleLinux7
supported_pg_type:
  - EPAS
  - PG
supported_pg_version:
  - 14
  - 15
  
 supported_pgd_version:
  - 5
```